README.md

```diff
@@ -64,7 +64,7 @@ Dispatch notifications are sent containing only the alert name, omitting additio
 - exports['ps-dispatch']:Fight()
 - exports['ps-dispatch']:FleecaBankRobbery(camId)
 - exports['ps-dispatch']:HouseRobbery()
-- exports['ps-dispatch']:HumaneRobery()
```

Comment:
Unfortunately, this change is backward incompatible and will break scripts that rely on this export if they upgrade to a newer version. We can't rename this unless we make a new major release. Please revert this change.